src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -66,19 +68,15 @@ internal void LogDiagnostic(Diagnostic diagnostic, CultureInfo culture)
                 message = "<None>";
```

Comment:
I had the same thought and had already scrapped it in a change I hadn't yet pushed.